tests/datatypes/test_set.py

```diff
@@ -157,12 +157,25 @@ def test_difference_update(self):
             print(x)
             """)
 
-    def test_intersection_update(self):
+    def test_issuperset(self):
         self.assertCodeExecution("""
             x = {1, 2, 3}
-            y = {3, 4, 5}
-            x.intersection_update(y)
-            print(x)
```

Comment:
This looks like an unintended change, it is deleting the intersection_update test -- can you bring it back please?